test/JDBC/expected/parallel_query/binary-index-vu-prepare.out

```diff
@@ -0,0 +1,68 @@
+create table tab_binary (a binary(4), b int)
```

Comment:
why diff expected file for parallel query?

Reply:
Yes, this file shouldn't have a output diff for parallel query. removing it. thanks.